File: app/src/main/java/lemond/annoying/gamerscompanion/fragment_news/view/ViewMoreNewsViewHolder.java
package lemond.annoying.gamerscompanion.fragment_news.view;

import android.support.v7.widget.RecyclerView;

import lemond.annoying.gamerscompanion.core.viewmodel.ViewMoreViewModel;
import lemond.annoying.gamerscompanion.databinding.ListItemViewMoreBinding;


public class ViewMoreNewsViewHolder extends RecyclerView.ViewHolder {

    private ListItemViewMoreBinding binding;

    public ViewMoreNewsViewHolder(ListItemViewMoreBinding binding) {
        super(binding.getRoot());
        this.binding = binding;
    }

    public void bindViewMoreViewModel(ViewMoreViewModel viewMoreViewModel) {
        binding.setViewModel(viewMoreViewModel);
        binding.executePendingBindings();
    }
}

File: app/src/main/java/lemond/annoying/gamerscompanion/main_activity/fragment_news/view/ViewMoreNewsViewHolder.kt
package lemond.annoying.gamerscompanion.main_activity.fragment_news.view

import android.support.v7.widget.RecyclerView

import lemond.annoying.gamerscompanion.core.viewmodel.ViewMoreViewModel
import lemond.annoying.gamerscompanion.databinding.ListItemViewMoreBinding


class ViewMoreNewsViewHolder(private val binding: ListItemViewMoreBinding) : RecyclerView.ViewHolder(binding.root) {

    fun bindViewMoreViewModel(viewMoreViewModel: ViewMoreViewModel) {
        binding.viewModel = viewMoreViewModel
        binding.executePendingBindings()
    }
}
